test/core/test/jest-expect.test.ts

```diff
@@ -636,7 +685,7 @@ describe('toSatisfy()', () => {
         }),
       )
     }).toThrowErrorMatchingInlineSnapshot(
-      `[AssertionError: expected Error: 2 to match object { message: toSatisfy{…} }]`,
+      `[AssertionError: expected Error: 2 to match object { Object (message) }]`,
```

Comment:
Not sure about this

Reply:
Yeah, this looks puzzling, but what's happening is we have truncation logic here for main error message and that just kicked in due to `toSatisfy` format got a bit longer. This is fine as well.